### PR TITLE
FIX: non-responsive checkbox after text

### PR DESCRIPTION
**Pull request by mr-vaibh**
_Please check the following:_

- [x] Do the tests still pass?[^1]
- [x] Does the code comply with the style guide? 
    - [x] Run `make lint` from the Wagtail root. 
- [x] For Python changes: Have you added tests to cover the new/fixed behaviour?
- [x] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    - [x] **Please list the exact browser and operating system versions you tested**:
    - [x] **Please list which assistive technologies [^3] you tested**: 
- [x] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**. 

I noticed that the checkboxes were not styled in the way they should
ideally, the text should be right next to the box

as a man of OCD, I've proposed these changes

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,8 @@ python manage.py createsuperuser
 python manage.py runserver
 ```
 
-For detailed installation and setup docs, see [docs.wagtail.org](https://docs.wagtail.org/).
+Note: Above commands are the implementation of Wagtail, NOT setup guide.
```
Comment (lb-) [inline README.md:44]: Please can this unrelated change be removed from this PR - I think you have already covered this suggestion in a seperate PR.

**Comment by lb-** [comment]
@mr-vaibh please can you add some clarity as to what you mean by 'some devices' - also maybe a screenshot of before / after will help.

File: wagtail/admin/static_src/wagtailadmin/css/normalize.css [hunk 2]
```diff
@@ -450,7 +450,7 @@ input[type="submit"] {
 input[type='checkbox'],
 input[type='radio'] {
   box-sizing: border-box; /* 1 */
-  padding: 0; /* 2 */
+  padding: 0 5px; /* 2 */
```
Comment (lb-) [inline wagtail/admin/static_src/wagtailadmin/css/normalize.css:453]: This padding makes hard assumption about the zoom and font size. 

Maybe we should explore a more robust solution. 
em/rem units could help. 

Also. It might be worth fixing the checkbox heights to a similar approach.  

Please also check other styles for checkbox styling. This is normalize so there may be widget specific styles to change instead of.

Comment (mr-vaibh) [reply wagtail/admin/static_src/wagtailadmin/css/normalize.css:453]: That makes sense. Okay, so I'll work on relative padding/styling and some other minor changes 👍

File: README.md (hunk 1 above)
Comment (mr-vaibh) [reply README.md]: yes, you're right

It was my bad, I've reverted this specific commit.

**Comment by lb-** [comment]
@mr-vaibh thanks for this but some major re-work has been done for the page editor and all fields for 4.0
From what I can see this is now fixed as part of that work.

Screen Shot 2022-08-12 at 5 34 28 pm

If you still see a problem on the latest main branch can you raise a new bug for it and you can help with a fix or we will aim to fix before 4.0 goes live.

You can see the PR that has the field changes here https://github.com/wagtail/wagtail/pull/8921
